Readme.md

```diff
@@ -1,69 +1,133 @@
-BroadGauge
-==========
+# BroadGauge
 
-Platform for managing training/workshops by connecting interested trainers and educational institutes interested in conducting trainings for their students. 
+A platform for managing training sessions and workshops to connect trainers and educational institutes interested in conducting training for their students. 
 
-This is going to be used for managing Python workshops going to be conducted across Ireland.
+This project is a fork of [Anand Chitipothu](https://github.com/anandology)'s
+ [broadgauge](https://github.com/anandology/broadgauge) project.
 
+## Requirements
 
+See [`requirements.txt`](https://github.com/PythonIreland/broadgauge/blob/master/requirements.txt) for the full set of 
+requirements.
 
-Requirements
-------------
-
-* PostgreSQL
 * Python 2.7
+* PostgreSQL
 * virtualenv
 
-How to setup
-------------
+# How to setup
+
+The short version:
+
+* Install dependencies system-wide
+* Clone the GitHub repo
+* Create the Python virtualenv
+* Install project-specific dependencies in the virtualenv
+* Configure the database
+  * Create a new database
+  * Create an account for use by the BroadGauge server
+  * Import the database schema as that user
+* Start the BroadGauge server
+
+## Install the dependencies
+
+### Linux
+
+There are two stages to installing the dependencies; some you'll install system-wide and some you'll install in the project's Python virtual environment. The former set of dependencies guarantee the latter set can be installed successfully.
+
+Using your package manager, install the following dependencies system-wide:
+
+    python
+    python-dev
+    postgresql
+    postgresql-server-dev-9.3
```

Comment:
Minor nitpick: people on different linux distros may have 9.2 or 9.4 PostGres installed, so if you could phrase this to avoid the explicit version dependence, it would be great.